cypress/integration/WebInterface/Measure/EditMeasure/EditMeasureAddMetaData.spec.ts

```diff
@@ -16,7 +16,7 @@ describe('Edit Measure: Add Meta Data', () => {
         OktaLogin.Login()
```

Comment:
we should not need log in and log out, these lines c an be deleted if we are seeding data with the api

Reply:
unnecessary logins and logouts have been removed